src/variables/components/TypeAheadVariableDropdown.tsx

```diff
@@ -284,6 +284,7 @@ class TypeAheadVariableDropdown extends PureComponent<Props, MyState> {
         testID={this.props.testID || `variable-dropdown--${name}`}
         onClickAway={this.onClickAwayHere}
         menuOpen={menuOpen}
+        customFocus
```

Comment:
Should we rename this to `disableAutoFocus` or something, @daviddkkim?

Reply:
yeah I think that'd be the better name. Let's get around to that after we finish fixing the tests